api/src/main/resources/MedicalHistoryField.hbm.xml

```diff
@@ -13,7 +13,12 @@
 		
 		<property name="name" type="java.lang.String" length="255" not-null="true"/>
 		
-		<property name="tabIndex" type="int" />
+		<!-- tabIndex property in database is mapped to tab_index.
+		When sqldiff.xml was used it was stored as tabIndex in db. 
+		This caused problem in PostgreSQL because Postgres converts all column 
+		names to lower case when fetching from db and thus no result was returned since
+		column tabIndex was being searched as tabindex and thus was never found -->
+		<property name="tabIndex" column="tab_index" type="int" />
```

Comment:
What happens to those who already had the old column name?

Reply:
That will be a problem. Thanks for pointing that out. Let me see if I can find another solution for it.